src/main/java/org/apache/sling/feature/builder/BuilderContext.java

```diff
@@ -64,6 +64,18 @@
     /** Used in override rule to match all coordinates */
     public static final String COORDINATE_MATCH_ALL = "*";
 
+    /** Used to handle configuration merging - fail the merge when there is a clash for a PID - this is the default */
+    public static final String CONFIG_FAIL_ON_CLASH = "CLASH";
+
+    /** Used to handle configuration merging - fail the merge only when there is a clash on a property level */
+    public static final String CONFIG_FAIL_ON_PROPERTY_CLASH = "PROPERTY_CLASH";
+
+    /** Used to handle configuration merging - use the latest configuration, but don't merge */
+    public static final String CONFIG_USE_LATEST = "USE_LATEST";
```

Comment:
Is there also an opposite of this, in case you want to force not to use the latest be keep the 'oldest'?

Reply:
Merge the features the other way around :-)

Reply:
Fair enough :)

Reply:
I added a USE_FIRST and MERGE_FIRST policy :-)